src/svelte/Components/Modals/AddMacro.svelte

```diff
@@ -0,0 +1,82 @@
+<script>
+  export let addMacro;
+  let name = "";
+  let befehl = "";
+  let error = false;
+
+  function close() {
+    document.getElementById("addMacroModal").click();
+    name = "";
+    befehl = "";
+    error = false;
+  }
+
+  function submit(e) {
+    e.preventDefault();
+    if (name.trim() === "") {
+      error = true;
+      return;
+    }
+    addMacro(name, befehl);
+    close();
+  }
+</script>
+
+<input type="checkbox" id="addMacroModal" class="modal-toggle" />
+<label class="modal cursor-pointer">
+  <label for="" class="modal-box relative flex flex-col space-y-2">
```

Comment:
`for=""` kann weg

Reply:
Habs erstmal weg gemacht und so kommentiert:
![image](https://user-images.githubusercontent.com/77924509/212128415-1064bee4-dd5e-4c00-908f-ff725c8327b5.png)